src/scratch/UCERF3/FaultSystemRupSet.java

```diff
@@ -890,7 +890,7 @@ public void buildClusterRups(RuptureConnectionSearch search) {
 		setClusterRuptures(ruptures);
 	}
 	
-	private static class ClusterRupCalc implements Callable<ClusterRupture> {
+	public static class ClusterRupCalc implements Callable<ClusterRupture> {
```

Comment:
How are you using this? Can just write cluster ruptures to JSON. After building a rupture set, call setClusterRuptures(ruptures) and then when you write it out to a zip file it will save all of that information so that, after loading from a zip file, you can call getClusterRuptures()

Reply:
Reverted . 

And your comments about get/setClusterRuptures were relevant to InversionFaultSystemRupSet.java changes below.